docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -277,6 +284,74 @@ cluster. Likewise, a root cluster will not enforce device trust on resources in
 leaf clusters.
 </Admonition>
 
+## Optional: Role-based trusted device enforcement
+
+Role-based configuration enforces trusted device access at the role level. It
+can be configured with the `spec.options.device_trust_mode` (`required`,
+`optional`, `off`) option and applies to the resources in its `allow` rules. It
+works similar to [`require_session_mfa`](./per-session-mfa.mdx).
+
+To enforce authenticated device checks for a specific role, update the role with the following:
+
+```diff
+kind: role
+version: v6
+metadata:
+  name: example-role
+spec:
+  options:
+    # require authenticated device check for this role
++   device_trust_mode: "required" # add this line
+  allow:
+    logins: ['admin']
+    kubernetes_groups: ['edit']
+    node_labels:
+      '*': '*'
+    ...
+
+```
+
+<Admonition type="tip" title="Per-role vs. Cluster-wide">
+Per-role configuration let's you progressively block unauthorized device access,
+and is useful for getting started with Teleport Device Trust.
+
+Once all the devices used by engineers are enrolled in Teleport, cluster-wide
+configuration will ensure every resources can only ever be accessed with a
+trusted device.
+</Admonition>
+
+## Optional: Locking a trusted device
+
+Similar to [session and identity locking](./locking.mdx), a device can be locked
+using `tctl lock`.
+
+Locking blocks certificate issuance and ongoing or future accesses originating
+from a locked device. Locking a device only works if device trust is enabled and
+if the device is enrolled to Teleport.
+
+Find a device ID to lock:
+
+```code
+$ tctl devices ls
+Asset Tag     OS    Enroll Status   Device ID
+------------ -----  ------------- ------------------------------------
+(=devicetrust.asset_tag=)  macOS  enrolled      9cdfc0ad-64b7-4d9c-9342-50e97f418ba0
```

Comment:
I would if we should create a variable for the device ID, for consistency sake? WDYT?

Reply:
I think thats a good idea. Updating this one with variable.